src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/AbstractServerTask.groovy

```diff
@@ -73,15 +73,15 @@ abstract class AbstractServerTask extends AbstractTask {
     }
 
     protected File getServerDir(Project project){
-        String serverDirectory
-        if(server.outputDir !=null && !server.outputDir.isEmpty()){
-            serverDirectory = server.outputDir
-        }
-        else{
-            serverDirectory = getUserDir(project).toString() + "/servers/" + server.name
-        }
+        return new File(getUserDir(project).toString() + "/servers/" + server.name)
```

Comment:
a lot was removed and I can't remember why.  Can we talk through this one tomorrow?